.editorconfig

```diff
@@ -0,0 +1,11 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
```

Comment:
@vchepeli http://editorconfig.org/

@witmicko have we agreed upon this as the editorconfig for raincatcher? could be nice to add them to all repos through an Enhancement JIRA. In the monorepo it could be on the top level.